plugin/src/main/java/hudson/plugins/findbugs/FindBugsPublisher.java

```diff
@@ -154,6 +164,21 @@ public boolean isRankActivated() {
     }
 
     /**
+     * Added to properly uncoercing.
+     */
+    public boolean getIsRankActivated() {
+        return isRankActivated;
```

Comment:
this is the wrong name for this getter.
It should be `isIsRankActivated()` (sic).
I would recommend you rename this field so its called rankActivated - and then have a more sane name.